code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -376,17 +376,6 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			see_reagents = TRUE
 
 		if("Security")
-			if(!weapons_unlock)
-				var/count_secborgs = 0
-				for(var/mob/living/silicon/robot/R in GLOB.alive_mob_list)
-					if(R && R.stat != DEAD && R.module && istype(R.module, /obj/item/robot_module/security))
-						count_secborgs++
-				var/max_secborgs = 2
-				if(GLOB.security_level == SEC_LEVEL_GREEN)
-					max_secborgs = 1
-				if(count_secborgs >= max_secborgs)
-					to_chat(src, "<span class='warning'>There are too many Security cyborgs active. Please choose another module.</span>")
-					return
 			module = new /obj/item/robot_module/security(src)
```

Comment:
If youre doing this, do it properly or not at all

Reply:
I am, this is just removal of what will be unused code.
This is just a check to see dead, emagged and ert sec borgs for them to not count to the limit of available sec borgs on station.

Reply:
I mean from 379 downwards, unless you plan to remove that later on, in which case I ask "why not just in the original PR"

Reply:
Ah, because ERT borg is a thing, they can choose what skin too choose between security, medical and engineering, as long as they are not forced into the security module.

Reply:
Thats unrelated. This entire proc is the handler from pressing `Security` in the popup, which no longer exists.

Remove this entire switch statement piece.